src/app/build.gradle

```diff
@@ -16,6 +16,12 @@ jacocoTestReport {
 		xml.enabled = true
 		csv.enabled = false
 	}
+
+	afterEvaluate {
+		classDirectories = files(classDirectories.files.collect {
```

Comment:
Not sure what's going on here...?

Reply:
I actually just copied this syntax from your exclusion of the SparkWrapper from test coverage in OW. (I didn't want it to complain about missing coverage for the API client, although once integration tests are in we will get coverage)

Reply:
Although actually, not sure it's working! Codecov still seems to be complaining?

Reply:
Oh yeah, I've a feeling that didn't really work for me either, but I left it as a future exercise to look at.